rotkehlchen/tests/unit/test_stats.py

```diff
@@ -2,21 +2,12 @@
 
 import pytest
 
-from rotkehlchen.chain.ethereum.modules.compound.compound import Compound
 from rotkehlchen.chain.ethereum.modules.liquity.trove import Liquity
-from rotkehlchen.constants import ONE
-from rotkehlchen.constants.assets import A_COMP, A_CUSDC, A_DAI, A_USDC, A_WBTC
```

Comment:
`A_CUSDC` is used only on loopring and tests. Wish we could remove it from the constants :/